src/com/dirj/DirJ.java

```diff
@@ -3,38 +3,73 @@
 import java.io.File;
 import java.text.NumberFormat;
 import java.text.SimpleDateFormat;
-import java.util.Date;
-import java.util.Locale;
+import java.util.*;
 
 public class DirJ {
     private static final String DIR = "  <DIR>";
+    private static boolean onlyFiles = false;
+    private static boolean lowerCase = false;
+    private static boolean helpOption = false;
+
     public static void main(String[] args) {
+        for (String arg : args) {
+            switch (arg) {
+                case "/B":
```

Comment:
Lets mimic classic windows `dir` behaviour: uppercase and lowercase arguments should both work in the same way. 

Note: unix-like operation systems (linux, mac OS, etc) usually distinguish upper and lower case characters.